plugins/base/src/main/kotlin/renderers/html/HtmlRenderer.kt

```diff
@@ -90,17 +90,17 @@ open class HtmlRenderer(
                     listOf(
                         BasicTabbedContentType.CONSTRUCTOR,
                         BasicTabbedContentType.TYPE,
-                        BasicTabbedContentType.FUNCTION,
-                        BasicTabbedContentType.PROPERTY
+                        BasicTabbedContentType.PROPERTY,
+                        BasicTabbedContentType.FUNCTION
                     )
                 ),
             if (extensions.isEmpty()) null else ContentTab(
                 "Members & Extensions",
                 listOf(
                     BasicTabbedContentType.CONSTRUCTOR,
                     BasicTabbedContentType.TYPE,
-                    BasicTabbedContentType.FUNCTION,
                     BasicTabbedContentType.PROPERTY,
+                    BasicTabbedContentType.FUNCTION,
```

Comment:
Would be nice to drop a comment about the order's meaning and why the order is the way it is

Reply:
Unfortunately, here the order doesn't make any difference, even though I expected it would :( Here it's purely cosmetic and I made the change for consistency. Something tells me leaving a comment like that here could be confusing down the road..

I'll write the tests for the true order though